src/components/common/Input/index.tsx

```diff
@@ -1,14 +1,22 @@
 import { InputPropsType } from 'type/common/input.type';
 import * as S from './style';
 
-const Input = ({ desc, placeholder, type, name, onChange }: InputPropsType) => {
+const Input = ({
+  desc,
+  placeholder,
+  type,
+  name,
+  value,
+  onChange,
+}: InputPropsType) => {
   return (
     <div>
-      <S.Desc>{desc}</S.Desc>
+      {desc === undefined ? '' : <S.Desc>{desc}</S.Desc>}
```

Comment:
```suggestion
      {desc ? <S.Desc>{desc}</S.Desc> : ''}
```

이게 더 깔끔할 것 같은데, 동작을 할지 모르겠네여 확인 해주세영